control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -743,6 +755,64 @@ impl TryFrom<get_volumes_request::Filter> for Filter {
     }
 }
 
+/// Trait to be implemented for CreateVolume operation.
+pub trait CreateSnapshotCloneInfo: Send + Sync + std::fmt::Debug {
+    /// Get the clone source snapshot uuid.
+    fn source_snapshot(&self) -> &SnapshotId;
+    /// Get the generic volume parameters.
```

Comment:
`Get the generic volume create parameters.`

Reply:
Done